packages/eds-core-react/src/components/SideSheet/SideSheet.stories.tsx

```diff
@@ -43,3 +78,200 @@ export const Default: Story<SideSheetProps> = (args) => {
+  const [toggle, setToggle] = useState(false)
+  const [value, updateValue] = useState([1, 50])
+  const changeHandler = (
+    event: FormEvent<HTMLDivElement>,
+    value: number[] | number,
+  ) => {
+    updateValue(value as number[])
+  }
+
+  const UnstyledList = styled.ul`
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+  `
+
+  return (
+    <Container>
+      <TopBar>
+        <TopBar.Header>Files</TopBar.Header>
+        <TopBar.CustomContent>
+          <Search
+            aria-label="sitewide"
+            id="search-normal"
+            placeholder="Search"
+          />
+        </TopBar.CustomContent>
+        <TopBar.Actions>
+          <Icons>
+            <Icon name="account_circle" size={16} title="user" />
+            <Icon name="notifications" size={16} />
+            <Button variant="ghost_icon" onClick={() => setToggle(!toggle)}>
+              <Icon name="filter_list" size={16} />
+            </Button>
+          </Icons>
+        </TopBar.Actions>
+      </TopBar>
+      <Table>
+        <Table.Head>
+          <Table.Row>
+            {columns.map((col) => (
+              <Table.Cell key={`head-${col.accessor}`}>{col.name}</Table.Cell>
+            ))}
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>
+          {cellValues?.map((row) => (
+            <Table.Row key={row.toString()}>
+              {row.map((cellValue) => (
+                <Table.Cell key={cellValue}>{cellValue}</Table.Cell>
+              ))}
+            </Table.Row>
+          ))}
+        </Table.Body>
+      </Table>
+      <SideSheet open={toggle} onClose={() => setToggle(!toggle)}>
+        <div style={{ padding: '6px' }}>
+          <Typography variant="h4">Filters</Typography>
+          <br />
+          <Typography variant="h6" color="disabled">
+            Labels
+          </Typography>
+          <UnstyledList>
+            <li>
+              <Checkbox
+                label="Fruits"
+                name="multiple"
+                value="first"
+                defaultChecked
+              />
+            </li>
+            <li>
+              <Checkbox label="Vegetables" name="multiple" value="second" />
+            </li>
+            <li>
+              <Checkbox label="Favourites" name="multiple" value="third" />
+            </li>
+          </UnstyledList>
+          <br />
+          <Typography id="range-slider-label" variant="h6" color="disabled">
+            Price range
+          </Typography>
+          <Slider
+            value={value}
+            onChange={changeHandler}
+            ariaLabelledby="range-slider-label"
+          />
+          <p style={{ marginTop: '1.5rem' }}>
+            <small>$: {value.join(', ')}</small>
+          </p>
+        </div>
+      </SideSheet>
+    </Container>
+  )
+}
+
+export const Scrolling: Story<SideSheetProps> = () => {
+  const cellValues = toCellValues(data, columns)
+  const [toggle, setToggle] = useState(false)
```

Comment:
Suggest default state to be open so its clearer to users that this is about side sheet